fix building when asm/page.h is not available

http://bugs.gentoo.org/168278

the loop-aes patch adds this ...

--- mount/swapon.c
+++ mount/swapon.c
@@ -31,7 +31,9 @@
 #include <sys/ioctl.h>
 #include <sys/utsname.h>
 #include <sys/time.h>
+#ifdef HAVE_asm_page_h
 #include <asm/page.h>
+#endif
 #include "xmalloc.h"
 #include "swap_constants.h"
 #include "swapargs.h"
